desc/equilibrium/equilibrium.py

```diff
@@ -1770,24 +1795,10 @@ def optimize(
             verbose=verbose,
             maxiter=maxiter,
             options=options,
+            copy=copy,
         )
 
-        if verbose > 0:
-            print("Start of solver")
-            objective.print_value(objective.x(eq))
-            for con in constraints:
-                con.print_value(*con.xs(eq))
-        for key, value in result["history"].items():
-            # don't set nonexistent profile (values are empty ndarrays)
-            if value[-1].size:
-                setattr(eq, key, value[-1])
-        if verbose > 0:
-            print("End of solver")
-            objective.print_value(objective.x(eq))
-            for con in constraints:
-                con.print_value(*con.xs(eq))
-
-        return eq, result
+        return things[0], result
 
     def _optimize(  # noqa: C901
```

Comment:
why do we still carry this old optimize method around?

Reply:
we have an automated test for it for a basic case (solovev with target aspect ratio), we could look at getting rid of it in a future version, it would still be available in older versions that are referenced in the paper